src/main.ts

```diff
@@ -2,6 +2,8 @@ import {Aurelia} from 'aurelia-framework'
 import {PLATFORM} from 'aurelia-pal';
 import {I18N, Backend} from 'aurelia-i18n';
 import environment from './environment';
+import 'materialize-css';
+import 'aurelia-materialize-bridge';
```

Comment:
you are right, this might not be needed. that is just one of the "better have it there and if everything works, let's try without it" cases. I just have this code in my repo, which itself is working fine. So i thought.... could not harm :P